### DISCUSSION TITLE
intercept multiple URLs

### DISCUSSION THREAD

**Discussion by baptx** [mitmproxy/mitmproxy#5970] [General]
Hello, it is possible to intercept a specific request in mitmproxy by pressing `i` and then adding the URL like this:
`set intercept 'https://example.com/test'`
However it is not working if we add a second URL. I tried a space and comma as a separator but it did not work. I also tried to add another URL between single quotes but multiple arguments are not accepted. Using `&` like described on https://docs.mitmproxy.org/stable/mitmproxytutorial-interceptrequests/ did not work for this case either.
How can we do it?

Update: I found a solution while writing the question so I will share it below.

`set intercept ~u 'https://example.com/test|https://example.org/test'`

It was not the most intuitive solution so maybe allowing multiple arguments or the `&` instead of the regex could be useful to improve the user experience.

**Comment by ServeurpersoCom** [comment]
Thanks. I need to use replace multiple expression with --map-remote (-M)
I this it's like
|A|B|anotherA|anotherB

Edit no, solution is multiple -M :

mitmdump -v -M "|aaaaaa|bbbbbb" -M "|foo|bar"
